### PR TITLE
Fix macos build

### PR DESCRIPTION
**Pull request by cxong**

### CODE REVIEW

File: occ/CMakeLists.txt [hunk 1]
```diff
@@ -3,17 +3,20 @@ cmake_minimum_required(VERSION 3.2)
 project(occ)
 
 # debug flags (-fuse-old=gold is for Travis)
+if(APPLE)
```
Comment (cxong) [inline occ/CMakeLists.txt:6]: memory sanitizer doesn't support apple silicon

**Comment by gurka** [comment]
I will merge this since you already done the work, but I don't work on this project anymore so I won't merge any more PRs

**Comment by cxong** [comment]
Thanks! I plan to continue working on this project, towards implementing the whole game. Would it be ok to grant me write access? Otherwise I will need to work on a fork.

**Comment by gurka** [comment]
> Thanks! I plan to continue working on this project, towards implementing the whole game. Would it be ok to grant me write access? Otherwise I will need to work on a fork.

Cool! I sent you an invite

**Pull request merged**